### ISSUE TITLE
Is this working 

### ISSUE THREAD

**Issue by isAlmogK** [lmtm/node-talon#1]
Just tried using this but it's not really working I keep getting 

replyText=<div dir="ltr">aaaaa aaaa aaaaa<br clear="all"><div><br></div>-- <br><div class="gmail_signature" data-smartmail="gmail_signature"><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><div><div dir="ltr"><br><br style="font-size:12.8px"><table align="left" style="font-size:12.8px;font-family:arial"><tbody><tr><td valign="top" style="font-family:arial;font-size:small"><img src="https://docs.google.com/a/almog.io/uc?id=0B1wKmv1NGEdOZHJ4ekdZTjRjZ1E&amp;export=download"></td><td style="font-family:Arial,Helvetica,sans-serif;font-size:13px;color:black"><strong>?? Almog R&amp;D - Development Studio??</strong><br>????<a href="http://almog.io/" target="_blank">almog.io</a><br><br><strong>?? Almog Koren - CEO / Founder</strong><br>?? Mobile: +61 04 3461 2068 ??|?? LinkedIn:??<a href="http://www.linkedin.com/in/almogkoren" target="_blank"><font color="#225373">almogk</font>oren</a>????|?? Twitter:??<a href="http://twitter.com/IsAlmog" target="_blank"><font color="#225373">@</font>IsAlmog</a><br>?? Email:??<a href="mailto:almog@almog.io" style="color:rgb(34,83,115);text-decoration:none" target="_blank">almog@almog.io</a>??| Offical Meteor Partner<br><br>?? We just launched in private beta <a href="https://pipely.io/" target="_blank">Pipley</a> - The all-in-one platform for exhibitors.??</td></tr></tbody></table></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div> </div><div hspace="streak-pt-mark" style="max-height:1px"><img alt="" style="width:0px;max-height:0px;overflow:hidden" src="https://mailfoogae.appspot.com/t?sender=aYWxtb2dAYWxtb2cuaW8%3D&amp;type=zerocontent&amp;guid=1929dbd5-ca90-4725-8a73-97cdb6df5cc0"><font color="#ffffff" size="1">???</font></div>

**Comment by shyamal890** [comment]
Yes, I too was not able to make it work. It gives the same output as the input.
